internal/protoschema/plugin/pluginpubsub/pluginpubsub.go

```diff
@@ -48,5 +49,6 @@ func Handle(
 	}
 
 	responseWriter.SetFeatureProto3Optional()
+	responseWriter.SetFeatureSupportsEditions(descriptorpb.Edition_EDITION_2023, descriptorpb.Edition_EDITION_2023)
```

Comment:
Should the max be `Edition_EDITION_2024` or unbounded `Edition_EDITION_MAX`?

Reply:
We definitely should not set it to "max", because future edition semantics could definitely influence how we generate JSON schemas, and we also can't be certain that we can normalize future edition semantics when generating the schema for Pub/Sub.

It is probably safe to raise it to 2024. But I'd rather not just yet since there are no docs published yet for Edition 2024. Scanning through `descriptor.proto` and`*_features.proto`, looks like nothing that would impact this, but I don't know if the scope is completely defined yet. I saw a [GitHub issue](https://github.com/protocolbuffers/protobuf/issues/19955) that said they aren't targeting release until approximately Q3 (of 2025, counter-intuitively).